### DISCUSSION TITLE
job synchronization problem in Azuracast

### DISCUSSION THREAD

**Discussion by mundostreaming** [AzuraCast/AzuraCast#7036] [General]
hello. I have this job synchronization problem in Azuracast, how can you help me solve it please?

time="2024-03-31T17:03:37Z" level=info msg="read crontab: /etc/cron.d/azuracast"
time="2024-03-31T17:04:00Z" level=info msg=starting iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:05:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (1m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:06:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (2m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:07:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (3m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:08:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (4m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:09:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (5m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:10:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (6m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:11:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (7m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:12:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (8m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:13:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (9m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:14:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (10m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:15:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (11m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:16:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (12m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:17:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (13m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:18:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (14m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:19:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (15m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:20:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (16m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:21:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (17m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:22:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (18m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:23:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (19m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:24:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (20m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"
time="2024-03-31T17:25:00Z" level=warning msg="not starting: job is still running since 2024-03-31 17:04:00 +0000 UTC (21m0s elapsed)" iteration=0 job.command="azuracast_cli azuracast:sync:run" job.position=0 job.schedule="0 * * * * * *"

**Comment by BusterNeece** [comment]
See #7101 for an issue to investigate this.
